lib/turn.rb

```diff
@@ -1,2 +1,40 @@
 class Turn
+  attr_reader :player,
+              :round
+
+  def initialize(player, round)
+    @player = player
+    @round = [0..10]
+  end
+
+  def player_won_round
+    if @player.place_pins.results == [0, 1]
+      p "Look at you go!?"
+      p "-".times 10
+    end
+    if @player.place_pins.results == [0, 2]
+      p "You heating up but you won't do it again."
+      p "-".times 10
+    end
+    if @player.place_pins.results == [0, 3]
+      p "I let you have that one!"
+      p "-".times 10
+    end
+    if @player.place_pins.results == [0, 4]
+      p "Whatever...you win I guess. Best two out of three?!"
+    end
+  end
+
+  def player_lost_round
```

Comment:
not sure if we have a complete loss condition unless the player quits the game

Reply:
If you can't lose then what's the point of playing anything?